core/src/main/resources/swagger/api-docs.yaml

```diff
@@ -823,6 +816,7 @@ paths:
             - profile
   /api/admin/refreshToken/{userSubjectId}:
     delete:
+      deprecated: true
```

Comment:
refresh token endpoints are slated for removal here: https://github.com/broadinstitute/rawls/pull/1331

they're no longer supported. UIs and orchestration have all been stubbed out to not call these rawls endpoints so they're now safe to remove